packages/atlas/src/api/queries/fragments.graphql

```diff
@@ -282,6 +282,23 @@ fragment CommentFields on Comment {
   text
   status
   commentcreatedeventcomment {
-    inBlock
+    inExtrinsic
+  }
+}
+
+fragment MetaprotocolTransactionStatusEventFields on MetaprotocolTransactionStatusEvent {
+  inExtrinsic
+  inBlock
+  status {
+    __typename
+    ... on MetaprotocolTransactionErrored {
+      message
+    }
+    ... on MetaprotocolTransactionSuccessful {
+      dummy
+    }
+    ... on MetaprotocolTransactionPending {
+      dummy
+    }
```

Comment:
I guess we could just skip those? Since type information will be conveyed via `__typename` anyway